classes/Project/ProjectFactory.js

```diff
@@ -760,6 +790,8 @@ export default class ProjectFactory {
             label: annotationData.label,
             items: await this.getLines(annotationData),
             partOf: annotationData.partOf,
+            prev: annotationData?.prev?.startsWith(process.env.RERUMIDPREFIX) ? annotationData.prev : null,
```

Comment:
This check isn't needed, right? It is either already `null` or it is a resolvable value. Even if we haven't saved it to RERUM (which I think it always is here), the t-pen.org link will still provide what we expect.